doc/api/tty.md

```diff
@@ -17,50 +23,54 @@ $ node -p -e "Boolean(process.stdout.isTTY)" | cat
 false
 ```
 
-## Class: ReadStream
+In most cases, there should be little to no reason for an application to
+create instances of the `tty.ReadStream` and `tty.WriteStream` classes.
+
+## Class: tty.ReadStream
 <!-- YAML
 added: v0.5.8
 -->
 
-A `net.Socket` subclass that represents the readable portion of a tty. In normal
-circumstances, `process.stdin` will be the only `tty.ReadStream` instance in any
-Node.js program (only when `isatty(0)` is true).
+The `tty.ReadStream` class is a subclass of `net.Socket` that represents the
+readable side of a TTY. In normal circumstances `process.stdin` will be the
+only `tty.ReadStream` instance in a Node.js process.
 
-### rs.isRaw
+### readstream.isRaw
 <!-- YAML
 added: v0.7.7
 -->
 
-A `Boolean` that is initialized to `false`. It represents the current "raw" state
-of the `tty.ReadStream` instance.
+A `boolean` that is `true` if the TTY is currently configured to operate as a
+raw device. Defaults to `false`.
 
-### rs.setRawMode(mode)
+### readstream.setRawMode(mode)
 <!-- YAML
 added: v0.7.7
 -->
 
-`mode` should be `true` or `false`. This sets the properties of the
-`tty.ReadStream` to act either as a raw device or default. `isRaw` will be set
-to the resulting mode.
+* `mode` {boolean} If `true`, configures the `tty.ReadStream` to operate as a
+  raw device. If `false`, configures the `tty.ReadStream` to operate in its
+  default mode. The `readstream.isRaw` property will be set to the resulting
+  mode.
 
-## Class: WriteStream
+## Class: tty.WriteStream
 <!-- YAML
 added: v0.5.8
 -->
 
-A `net.Socket` subclass that represents the writable portion of a tty. In normal
-circumstances, `process.stdout` will be the only `tty.WriteStream` instance
-ever created (and only when `isatty(1)` is true).
+The `tty.WriteStream` class is a subclass of `net.Socket` that represents the
+writable side of a TTY. In normal circumstances, `process.stdout` and
+`process.stderr` will be the only `tty.WriteStream` instances created for a
+Node.js process.
 
 ### Event: 'resize'
 <!-- YAML
 added: v0.7.7
 -->
 
-`function () {}`
-
-Emitted by `refreshSize()` when either of the `columns` or `rows` properties
-has changed.
+The `'resize'` event is emitted whenever the either of the `writestream.columns`
+or `writestream.rows` properties have changed. The listener callback is not
+passed any arguments when called.
```

Comment:
nit: This is a bit hard to parse… how about `No arguments are passed to the listener callback when called.`?